.github/workflows/check-buildstatus.yml

```diff
@@ -17,5 +17,5 @@ jobs:
         dotnet-version: 7.0.x
     - name: Restore dependencies
       run: dotnet restore
-    - name: Build TwitchLib.Api
+    - name: Build TwitchLib.Clinet
```

Comment:
You fixed one typo and made a new one :D `Client` instead of `Clinet`. Happens to all of us.